lua/main_config.lua

```diff
@@ -40,21 +40,25 @@ function gatekeeper_init()
 	local gt_conf
 
 	if gatekeeper_server == true then
-		local n_lcores = 2
-		local gk_lcores_tbl =
+		local n_gk_lcores = 2
```

Comment:
Set `n_gk_lcores` with the following expression: `2 * staticlib.count_numa_nodes(numa_table)`, and add the following comment just above it:
```Lua
-- The following expression to set the number of GK block instances is a good recommendation, but it may not be optimal for all cases.
``` 
Break the comment into multiple lines as needed. And add the following protection afterwards:
```Lua
if n_gk_lcores <= 0 then
	error("No GK block allocated for Gatekeeper server")
end
```